app/assets/stylesheets/application.css

```diff
@@ -737,14 +737,16 @@ li .byline a.story_has_suggestions {
 	margin: 1em 0 1em 32px; /* matching div.detail */
 }
 
-li.story.hidden *:not(div, .dropdown_parent, .dropdown_parent > div *, .comments_label, .link, .tags, img),
-li.story.hidden .score {
+li.story.hidden {
 	opacity: 0.25;
 }
 
-ol.show_hidden li.story.hidden {
-	opacity: 1.0 !important;
+li.story.hidden:only-child,
+ol.show_hidden li.story {
+	opacity: 1.0;
 }
+
+
```

Comment:
I assure you there are no two consecutive lines in my PR. Not sure why GH shows it this way.